### ISSUE TITLE
hexLiterals: false positives on hex lits without letters

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#780]
Reproducer: any hex literal without letter digits after `0x`.
Examples:
```go
0x30
0x11
0x1130243
```
Negative tests suite should be extended.

**Comment by Quasilyte** [comment]
Okay, I'll do it.
